netlify.toml

```diff
@@ -20,22 +20,22 @@
   publish = "./dist/acrabadabra/"
 
   # Default build command.
-  command = "npm run build"
+  command = "yarn build"
```

Comment:
Actually you can delete the others, because this one is the default, the others are just for context replacement. Since they are the same, they are not really useful.

See [Netlify doc](https://docs.netlify.com/configure-builds/file-based-configuration/#sample-file) for ref